css.js

```diff
@@ -1,36 +1,137 @@
 var loader = require("@loader");
 
+var isNode = typeof process === "object" && {}.toString.call(process) ===
+	"[object process]";
+var importRegEx = /@import [^uU]['"]?([^'"\)]*)['"]?/g;
+var resourceRegEx =  /url\(['"]?([^'"\)]*)['"]?\)/g;
+	
+var waitSeconds = (loader.cssOptions && loader.cssOptions.timeout)
+	? parseInt(loader.cssOptions.timeout, 10) : 60;
+var noop = function () {};
+var onloadCss = function(link, cb){
+	var styleSheets = document.styleSheets;
+	var i = styleSheets.length;
+	while( i-- ){
+		if( styleSheets[ i ].href === link ){
+			return cb();
+		}
+	}
+	setTimeout(function() {
+		onloadCss(link, cb);
+	}, 10);
+};
+var isSafari5 = function() {
```

Comment:
I think we should exclude this userAgent stuff and stick to a simpler strategy like loadCSS does: https://github.com/filamentgroup/loadCSS/blob/master/src/loadCSS.js#L44

Do it the same for all browsers: add the onload event and meanwhile also poll for the styleSheets. Whichever happens first, remove the event listener and resolve.

In general you try to avoid userAgent sniffing in JavaScript if at all possible because it adds a big maintenance burden.